### DISCUSSION TITLE
lookup this.files 'System.Team'

### DISCUSSION THREAD

**Discussion by kahaboom** [rfennell/AzurePipelines#1414] [Q&A] [answered]
Hello @rfennell and community. 

If I am using different teams in the tasks (black,white..) in azure devops

Will the phrase {{lookup this.files 'System.Team'}} work for me to show the team where this task is ? 

And about Deployment section, is it possible to also write a deployment status for the task ?
<img width="362" alt="Screenshot 2022-12-05 at 15 51 40" src="https://user-images.githubusercontent.com/80584072/205655302-600ff112-252e-4bcf-82da-4cfe2a8e3177.png">

Was searching for this info here but didn't found any answer.. thanks

**Comment by rfennell** [comment, marked as answer]
You can see all the fields available on an object using the general form

```
{{#forEach this.workItems}}
{{json this.id}}
{{/forEach}}
```

But, I don't think the is a `System.Team` field to loop up. There no association built in between a WI and Team in Azure DevOps.

As to the second question, all you could do is see of the information you need is in the `builddetails` object (again using the form 

```
{{ json builddetails }}
```
